packages/x-date-pickers/src/AdapterLuxon/AdapterLuxon.test.tsx

```diff
@@ -169,4 +169,16 @@ describe('<AdapterLuxon />', () => {
       });
     });
   });
+
+  describe('isWithinRange functionality', () => {
+    const adapter = new AdapterLuxon({ locale: 'en-US' });
+    it('should be equal with values in different locale', () => {
```

Comment:
If possible, it would be great to move the test inside `describeGregorianAdapter` (there is a `describe` for `isWithinRange`).
That way we make sure that all adapters are coherent on this behavior.

We have a `adapterFr` object that is using the french locale so we should be able to use it to create the two dates and then use `adapter` to run `isWithingRange` with an `en-US` locale.

If I'm missing something and there is a technical limitation preventing us from doing it, it's of course not a deal breaker :ok_hand:

Reply:
Thanks for a great suggestion! 🙏 
Moved the test and tests it's reliability. 😉